### PR TITLE
[WIP]Rocks rock!

### PR DESCRIPTION
**Pull request by KazeEspada**
Implements rocks. New items that spawn on the asteroid so that you can stone someone to death or beat a man(or goliath) to death with a rock.

### CODE REVIEW

**Comment by Bawhoppen** [comment]
bro I got just the sprites for you

File: code/game/objects/items/weapons/twohanded.dm [hunk 1]
```diff
@@ -412,3 +412,15 @@
...
+	name = "rock"
+	desc = "If there were two miners were on the mining station and one of them killed the other with a rock, would that be fucked up or what?"
+	force_unwielded = 6
+	force_wielded = 12
+	icon = 'icons/obj/mining.dmi'
+	icon_state = "slag"
+	force = 6
+	w_class = 2
+	throwforce = 10
+	attack_verb = list("beat", "tnwomped", "stoned", "fucked up")
```
Comment (Jordie0608) [inline code/game/objects/items/weapons/twohanded.dm]: tnwomped

File: code/game/objects/items/weapons/twohanded.dm [hunk 2]
```diff
@@ -412,3 +412,15 @@
...
 			M.Copy_Parent(user, 100, user.health/2.5, 12, 30)
 			M.GiveTarget(L)
+
+/obj/item/weapon/twohanded/rock
+	name = "rock"
+	desc = "If there were two miners were on the mining station and one of them killed the other with a rock, would that be fucked up or what?"
+	force_unwielded = 6
+	force_wielded = 12
+	icon = 'icons/obj/mining.dmi'
+	icon_state = "slag"
```
Comment (KorPhaeron) [inline code/game/objects/items/weapons/twohanded.dm]: This is the worst sprite in the game, please no

Comment (KazeEspada) [inline code/game/objects/items/weapons/twohanded.dm]: Its a black lump. Do you seriously want HD rocks?

Comment (Bawhoppen) [inline code/game/objects/items/weapons/twohanded.dm]: Kor I didn't think I was on YOUR git ignore list too

Comment (KorPhaeron) [inline code/game/objects/items/weapons/twohanded.dm]: I commented before you did though

**Comment by KazeEspada** [comment]
Forgot, this is WIP

**Comment by Zilenan91** [comment]
Can you put rocks in sandbags to make bags of rocks that do more damage

**Comment by Bawhoppen** [comment]
sandbags dont do damage

**Comment by Zilenan91** [comment]
But what if the bag is full of rocks instead of sand and you hit people with it

File: code/game/objects/items/weapons/twohanded.dm (hunk 1 above)
Comment (optimumtact) [inline code/game/objects/items/weapons/twohanded.dm]: >he's never been tnwomped

:laughing:

**Comment by TechnoAlchemisto** [comment]
i have a 128x128 rock sprite if you want it

**Comment by MMMiracles** [comment]
do you find the rocks from newmans too or

**Comment by KazeEspada** [comment]
Im chopping up the rocks we have.

File: code/game/objects/items/weapons/twohanded.dm (hunk 2 above)
Comment (TheWiznard) [inline code/game/objects/items/weapons/twohanded.dm]: "If there were two miners _were_ on the mining station ... "
should be "If there were two miners on the mining station ... " 
no?

File: code/game/objects/items/weapons/twohanded.dm [hunk 3]
```diff
@@ -412,3 +412,15 @@
 			M.faction = user.faction.Copy()
 			M.Copy_Parent(user, 100, user.health/2.5, 12, 30)
 			M.GiveTarget(L)
+
+/obj/item/weapon/twohanded/rock
+	name = "rock"
+	desc = "If there were two miners were on the mining station and one of them killed the other with a rock, would that be fucked up or what?"
```
Comment (Incoming5643) [inline code/game/objects/items/weapons/twohanded.dm]: the "were" between "miners" and "on" is unnessiary and breaks flow with the meme your invoking.

Same for "of them" between "one" and "killed"

if there were two miners on the mining station and one killed the other with a rock, would that be fucked up or what?

Comment (KazeEspada) [inline code/game/objects/items/weapons/twohanded.dm]: Im leaving "of them" in to preserve the meme. But the extra were was removed.

File: code/game/objects/items/weapons/twohanded.dm [hunk 4]
```diff
@@ -412,3 +412,27 @@
...
+	name = "rock"
+	desc = "If there were two miners on the mining station and one of them killed the other with a rock, would that be fucked up or what?"
+	force_unwielded = 6
+	force_wielded = 12
+	icon = 'icons/obj/flora/rocks.dmi'
+	icon_state = "smallrock1"
+	force = 6
+	w_class = 2
+	throwforce = 10
+	attack_verb = list("beat", "tnwomped", "stoned", "fucked up")
```
Comment (RemieRichards) [inline code/game/objects/items/weapons/twohanded.dm]: tnwomped?

Comment (KazeEspada) [inline code/game/objects/items/weapons/twohanded.dm]: Its supposed to be thwomped

Comment (RemieRichards) [inline code/game/objects/items/weapons/twohanded.dm]: but it clearly isn't thwomped!

**Comment by Man-Shroom-** [comment]
6 damage is too little

**Comment by PKPenguin321** [comment]
>6 damage is too little

Their real damage is actually randomized from I think 6-8 and wielded goes up to 15

They also have a chance of being sharp

**Comment by Alek2ander** [comment]
Allahu akbar

**Comment by Supermichael777** [comment]
smoke those ash kababs

**Comment by Bawhoppen** [comment]
just close this it makes no sense why they are wieldable reeee

**Comment by TechnoAlchemisto** [comment]
Because they big

**Comment by Bawhoppen** [comment]
there are countless other things that should need to be wielded yet aren't, guns, tools, like nearly everything, rocks makes no sense to be wielded unless we make other stuff that should be wieldable

**Comment by TechnoAlchemisto** [comment]
rock heavy rock smash

**Comment by Bawhoppen** [comment]
fuck you bitch

**Comment by MMMiracles** [comment]
rock > bawhoppen

**Comment by KorPhaeron** [comment]
stale and conflicts, sounded fun though

**Comment by Bawhoppen** [comment]
I shall make proper rocks

**Pull request closed without merging**